src/Symfony/Component/VarDumper/Caster/StubCaster.php

```diff
@@ -31,8 +31,8 @@ public static function castStub(Stub $c, array $a, Stub $stub, $isNested)
             $stub->attr = $c->attr;
 
             if (Stub::TYPE_REF === $c->type && !$c->class && is_string($c->value) && !preg_match('//u', $c->value)) {
-                $stub->type = self::TYPE_STRING;
-                $stub->class = self::STRING_BINARY;
+                $stub->type = Stub::TYPE_STRING;
```

Comment:
This one looks like an actual bug fix however.

ping @nicolas-grekas: I didn't see any related test covering this.